support/doc/api/embeds.md

```diff
@@ -58,102 +58,122 @@ For example `https://my-instance.example.com/videos/embed/52a10666-3a18-4e73-93d
 
 Start the video at a specific time.
 Value must be raw seconds or a duration (`3m4s`)
+Default: starts at `0`
 
 ### stop
 
 Stop the video at a specific time.
 Value must be raw seconds or a duration (`54s`)
+Default: ends at content end
 
 ### controls
 
 Mimics video HTML element `controls` attribute, meaning that all controls (including big play button, control bar, etc.) will be removed.
 It can be useful if you want to have a full control of the PeerTube player.
 
 Value must be `0` or `1`.
+Default: `1`
 
 ### controlBar
 
 Hide control bar when the video is played.
 
 Value must be `0` or `1`.
+Default: `1`
 
 ### peertubeLink
 
 Hide PeerTube instance link in control bar.
 
 Value must be `0` or `1`.
+Default: `1`
 
 ### muted
 
 Mute the video by default.
 
 Value must be `0` or `1`.
+Default: `0`
```

Comment:
Default: `0` then tries to restore the last muted setting set by the user

Is this better ?

Reply:
The default is `tries to restore the last muted setting set by the user`. Because with the `0` value peertube just mutes the video, without trying to restore the user setting